src/content/docs/apm/agents/nodejs-agent/getting-started/monitor-your-nodejs-app.mdx

```diff
@@ -51,6 +51,14 @@ We have a few paths to install the Node.js agent, depending on what you're monit
 
     If you have a preference for manual installation via the command line, we currently have the option for [Legacy instrumentation for Lambda monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-legacy/#node).
   </Collapser>
+
+  <Collapser
+      id="azure"
+      title="Azure web app on Windows"
+  >
+    If you'd like to instrument an Azure Web App for your apps deployed as code in a Windows environment, you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure). 
```

Comment:
```suggestion
    If you'd like to instrument an Azure web app for your apps deployed as code in a Windows environment, you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure). 
```

Reply:
not sure if we're referring to azure web app in the general, like any web app, or as a specific product Azure offers "Azure Web App"